hasura.planx.uk/metadata/tables.yaml

```diff
@@ -589,6 +589,7 @@
     - role: demoUser
       permission:
         columns:
+          - analytics_link
```

Comment:
<img width="605" alt="image" src="https://github.com/user-attachments/assets/3abfb39c-54be-4f30-a18e-cb577db4e7d6">

Once I fixed the issue with missing column permissions on `operations`, I then hit this one.